lib/main_common.pm

```diff
@@ -1538,6 +1538,7 @@ sub load_extra_tests_console {
         loadtest 'console/mutt';
     }
     loadtest 'console/systemd_testsuite' if is_sle('15+') && get_var('QA_HEAD_REPO');
+    loadtest 'console/supportutils';
```

Comment:
is this supported in opensuse?

Reply:
should I go for testing also in openSUSE or let it run only on SLE? doesn't make much sense to run it on oS, I think.

Reply:
made it run only for SLE

Reply:
Why shouldn't it make sense to run it on oS? At best we can keep the same test coverage on both openSUSE and SLE so I would appreciate a lot if you can test it on openSUSE.